xtrack/base_element.py

```diff
@@ -17,20 +17,20 @@
 
 start_per_part_block = """
     {
-    const int64_t start_idx = part0->ipart; //only_for_context cpu_openmp
-    const int64_t end_idx = part0->endpart; //only_for_context cpu_openmp
+    const int64_t XT_part_block_start_idx = part0->ipart; //only_for_context cpu_openmp
+    const int64_t XT_part_block_end_idx = part0->endpart; //only_for_context cpu_openmp
 
-    const int64_t start_idx = 0;                                            //only_for_context cpu_serial
-    const int64_t end_idx = LocalParticle_get__num_active_particles(part0); //only_for_context cpu_serial
+    const int64_t XT_part_block_start_idx = 0;                                            //only_for_context cpu_serial
+    const int64_t XT_part_block_end_idx = LocalParticle_get__num_active_particles(part0); //only_for_context cpu_serial
 
     //#pragma omp simd // TODO: currently does not work, needs investigating
-    for (int64_t ii=start_idx; ii<end_idx; ii++) { //only_for_context cpu_openmp cpu_serial
+    for (int64_t XT_part_block_ii=XT_part_block_start_idx; XT_part_block_ii<XT_part_block_end_idx; XT_part_block_ii++) { //only_for_context cpu_openmp cpu_serial
```

Comment:
For good practice I would not start the name of a variable with a capital letter.

Reply:
I chose to prefix it as we do for the define flags (which are ofc all caps) to minimise the probability of having the same variable name, but feel free to name mangle it as you see fit; I have no strong opinion